packages/lib/src/instrumentation.ts

```diff
@@ -14,6 +14,8 @@ import {
 import { buildInfo, BuildInfo, recordBuildInfo } from "./buildInfo";
 import { HISTOGRAM_NAME } from "./constants";
 
+let IS_EAGERLY_PUSHED = false;
```

Comment:
Why the ALL_CAPS? That's usually used for constants, which this is not.

Reply:
my personal convention for local global variables — changing it!